puppet/modules/slave/manifests/init.pp

```diff
@@ -50,7 +50,6 @@
     group  => 'jenkins',
   }
   slave::pr_test_config { [
-    'foreman_ansible',
     'foreman_bootdisk',
```

Comment:
I think this and all other plugins you've converted should be moved.

Reply:
yeah as soon as i threw up the PR, i had just remembered those.

Reply:
updated